charts/enterprise/clusterissuer/docs/how-to.md

```diff
@@ -4,26 +4,17 @@ This guide will walk you through setting up `clusterissuer`, certificate managem
 
 ## Prerequisites
 
-Ensure you have the `enterprise` train enabled for `TrueCharts` as discussed [here](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts).
+- Ensure you have the `enterprise` train enabled for `TrueCharts` as discussed [here](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts).
+- Traefik is installed from enterprise train
 
-Ensure you have traefik installed, required for Ingress.
+## Set Scale Nameservers
 
-Search for clusterissuer in the `Apps` menu | `Available Applications` tab and click **Install**.
+It is important to configure Scale with reliable nameserver to avoid issues handling DNS-01 challenges. Under Network -> Global Configuration-> Nameservers, we recommend setting 1.1.1.1, 1.0.0.1 or 8.8.8.8.
```

Comment:
```suggestion
It is important to configure Scale with reliable nameserver to avoid issues handling DNS-01 challenges. Under Network -> Global Configuration-> Nameservers, we recommend setting 1.1.1.1, 9.9.9.9 or 8.8.8.8.
```